resources/mediaplayer/src/main/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackImpl.java

```diff
@@ -297,6 +272,92 @@ private void padding(byte[] data, int count) {
             data[i + offset] = paddingByte;
         }
     }
+    
+    /* Getters, Setters, Overridable methods*/
```

Comment:
Warning for these getters/setters: 
- do we really need all of them? why not access variables directly from within the class?
- careful with scope, you're breaking encapsulation.